meta-mbl-apps/recipes-support/spdlog/spdlog_%.bbappend

```diff
@@ -0,0 +1 @@
+SRCREV = "1549ff12f1aa61ffc4d9a8727c519034724392a0"
```

Comment:
Can you please add a comment clarifying the reason for picking this specific SHA?

Reply:
Added it when I fixed the licensing